Npm doesn't find git

Here my error with npm `v3.10.6`

```
npm ERR! not found: git
npm ERR! 
npm ERR! Failed using git.
npm ERR! This is most likely not a problem with npm itself.
npm ERR! Please check if you have git installed and in your PATH.
```

--- a/images/js/Dockerfile
+++ b/images/js/Dockerfile
@@ -3,7 +3,7 @@ FROM markadams/chromium-xvfb
 RUN apt-get update && apt-get install -y curl
 
 RUN curl -sL https://deb.nodesource.com/setup_6.x | bash -
-RUN apt-get install -y nodejs && npm install -g npm@latest
+RUN apt-get install -y nodejs git && npm install -g npm@latest
 
 WORKDIR /usr/src/app
 

